editor/code_editor.cpp

```diff
@@ -368,6 +368,12 @@ void FindReplaceBar::_update_results_count() {
 
 		int col_pos = 0;
 
+		if (is_ignoring_comments()) {
+			if (line_text.begins_with("#")) {
```

Comment:
This won't match for the majority of line comments because they don't begin with `#`, they begin with `\t#`

![image](https://user-images.githubusercontent.com/248383/197409125-765f5ac2-69c6-421a-b4bf-51eefb6f4f76.png)

Reply:
In reality this is also not language agnostic (It won't work for both GDScript and C#, for example).

Reply:
not sure if this is the problem because the lines are stripped in the begin and end, also other languages doesnt matter, because code editor is used for gdscript only, as far as I know!